lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -262,24 +280,40 @@ function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6)
   }
 
   foreach ($reportbackItems as $item) {
-    if ($item instanceof ReportbackItem) {
+    if (variable_get('rogue_collection', FALSE)) {
+      $item = (object) $item;
       $gallery['item_ids'][] = $item->id;
+      $gallery['items'][] = paraneue_dosomething_get_themed_reportback_item($item, TRUE);
     }
+    else {
+      if ($item instanceof ReportbackItem) {
+        $gallery['item_ids'][] = $item->id;
+      }
 
-    $gallery['items'][] = paraneue_dosomething_get_themed_reportback_item($item);
+      $gallery['items'][] = paraneue_dosomething_get_themed_reportback_item($item);
+    }
   }
 
   // Get total of Promoted & Approved RB Items prefetched.
   $gallery['prefetched'] = count($gallery['item_ids']);
 
-  // Get total of Promoted & Approved RB Items for Campaign.
-  $gallery['total_sum'] = $gallery['total_promoted'] + $gallery['total_approved'];
+  if (variable_get('rogue_collection', FALSE)) {
+    $gallery['total_sum'] = $gallery['total_approved'];
+  }
+  else {
+    // Get total of Promoted & Approved RB Items for Campaign.
+    $gallery['total_sum'] = $gallery['total_promoted'] + $gallery['total_approved'];
+  }
 
   // Get total remaining after subtracting the RB Items already retrieved.
   $gallery['remaining'] = $gallery['total_sum'] - $gallery['prefetched'];
 
   global $user;
 
+  // @TODO: how do we deal with the below?
```

Comment:
Hmm, good catch. The most responsible thing is probably setting up feature flags so that those calls can be made to Rogue (to both fetch reactions & save new ones). I also wonder if we can just remove reactions from old Phoenix, and _just_ have that as a feature on the new campaign template...?